README.rst

```diff
@@ -0,0 +1,175 @@
+django-ecsmanage
```

Comment:
I didn't really want to do this, but it allows `setup.py check` to work.

See: https://github.com/pypa/warehouse/issues/4079